com.github.gijsgoudzwaard.image-optimizer.json

```diff
@@ -1,19 +1,53 @@
 {
     "id": "com.github.gijsgoudzwaard.image-optimizer",
     "base": "io.elementary.BaseApp",
-    "base-version": "circe-24.08",
-    "runtime": "org.gnome.Platform",
-    "sdk": "org.gnome.Sdk",
-    "runtime-version": "49",
+    "base-version": "circe-25.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "25.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.vala"
+    ],
+    "build-options": {
+        "prepend-path": "/usr/lib/sdk/vala/bin/",
+        "prepend-ld-library-path": "/usr/lib/sdk/vala/lib"
+    },
     "command": "com.github.gijsgoudzwaard.image-optimizer",
     "finish-args": [
         /* X11 + XShm */
-        "--share=ipc", "--socket=fallback-x11",
+        "--share=ipc",
+        "--socket=fallback-x11",
         /* Wayland */
         "--socket=wayland",
         "--filesystem=home"
     ],
+    "cleanup": [
+        "/bin/sassc",
+        "/include",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/gir-1.0",
+        "/share/man",
+        "/share/vala"
+    ],
     "modules": [
+        {
+            "name": "libgee",
+            "build-options" : {
+                "make-install-args" : [
+                    "girdir=/app/share/gir-1.0",
+                    "typelibdir=/app/lib/girepository-1.0"
+                ]
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.8.tar.xz",
+                    "sha256": "189815ac143d89867193b0c52b7dc31f3aa108a15f04d6b5dca2b6adfad0b0ee"
+                }
+            ]
+        },
```

Comment:
I wonder whether this should be included in the elementary BaseApp. What do you think?

Reply:
Yes, so I suppose we may want to add libgee in the elementary BaseApp instead of here because it's commonly used in projects that use Vala.

Reply:
The elementary BaseApp now comes with libgee, so we can remove this from here.